clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -475,11 +500,18 @@ public Node leastLoadedNode(long now) {
     }
 
     public static AbstractResponse parseResponse(ByteBuffer responseBuffer, RequestHeader requestHeader) {
+        return parseResponseMaybeUpdateThrottleTimeMetrics(responseBuffer, requestHeader, null, 0);
```

Comment:
This is called by SaslClientAuthenticator. So, it's useful to reflect the metric on the current time window. Perhaps we can just use SystemTime.milliseconds() instead of 0?